gbpservice/nfp/orchestrator/coal/networking/nfp_gbp_network_driver.py

```diff
@@ -59,3 +65,8 @@ def set_promiscuos_mode(self, token, port_id):
         self.network_handler = openstack_driver.NeutronClient(self.config)
         super(NFPGBPNetworkDriver, self).set_promiscuos_mode(token, port_id)
         self.network_handler = openstack_driver.GBPClient(self.config)
+
+    def set_promiscuos_mode_v1(self, token, port_id):
```

Comment:
Why is this needed ?
We already have set_promiscuos_mode method. Where GBP PT id is passed.
In GBP work flow always pass PT ID